e2e/favorites.spec.ts

```diff
@@ -31,10 +31,11 @@ async function expectProjectPageLoaded(
 ) {
   await page.waitForURL(entityUrlPathname(projectId))
   await expect(page.getByRole('heading', { name: projectName })).toBeVisible()
-  await page.waitForURL(
-    `${entityUrlPathname(PUBLIC_PROJECT.id)}/discussion/default`,
-  )
-  await expect(page.getByRole('heading', { name: 'Discussion' })).toBeVisible()
+  await expectDiscussionPageLoaded(page, projectId)
```

Comment:
Wait for more elements to load on the discussions page, rather than only the URL and heading